rules-tests/Php80/Rector/FunctionLike/MixedTypeRector/Fixture/demo_file.php.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rector\Tests\Php80\Rector\FunctionLike\MixedTypeRector\Fixture;
+
+final class DemoFile
```

Comment:
please rename the fixture class, eg: `OnConstructorPromotion`, and rename file fixture,eg: `on_construction_promotion.php.inc`